src/main/java/edu/harvard/iq/dataverse/DatasetLinkingServiceBean.java

```diff
@@ -60,31 +42,15 @@
 
     public List<Dataverse> findLinkingDataverses(Long datasetId) {
         List<Dataverse> retList = new ArrayList<>();
-        for (DatasetLinkingDataverse dld : findDatasetLinkingDataverses(datasetId)) {
-            retList.add(dld.getLinkingDataverse());
+        Query query = em.createQuery("select object(o) from DatasetLinkingDataverse as o where o.dataset.id =:datasetId order by o.id");
```

Comment:
Any reason this wasn't added as a named query?

Reply:
No, just because I was only changing the ones related specifically to saved search. I did think about it. And sure, I can add the others too, while I'm at it.